File: app/src/at-vinhnguyen/java/asiantech/internship/summer/javacore/excercise7/Fibonacci.java
package asiantech.internship.summer.javacore.excercise7;

import java.util.Scanner;

public class Fibonacci {

    public static void main(String[] args) {
        System.out.print("Nhap n:");
        int n = new Scanner(System.in).nextInt();
        System.out.println("So fibonacci thu " + n + " la: " + getFibonacci(n));
    }

    private static int getFibonacci(int n) {
        if (n == 0 || n == 1) {
            return n;
        }
        return getFibonacci(n - 1) + getFibonacci(n - 2);
    }
}

File: app/src/at-vinhnguyen/java/asiantech/internship/summer/exercise7/Fibonacci.kt
package exercise7

import getFibonacciInPosition
import inputIntNumber

object Fibonacci {

    @JvmStatic
    fun main(args: Array<String>) {
        print("Nhap n:")
        val n = inputIntNumber()
        println("So fibonacci thu " + n + " la: " + n.getFibonacciInPosition())
    }
}
